src/shared/modules/favorites/staticScripts.js

```diff
@@ -19,113 +19,151 @@
  */
 
 export const scripts = [
+  {
+    folder: 'basics',
+    content: '// Connect to database\n:server connect',
```

Comment:
👍 
I think this should be "Connect to DBMS". We use "database" here and there but we should move away from that.

Reply:
OK, I will change the wording and add some tests :)